app/components/login_button_component.html.erb

```diff
@@ -0,0 +1,8 @@
+<%= content_tag(
+      :button,
+      **tag_options,
+      class: css_class,
+    ) do %> 
+  <%= content %>
+  <%= image_tag(asset_url(logo_path), width: width, height: height, alt: t('components.login_button.image_alt'), class: 'display-inline text-middle margin-left-1 margin-top-neg-05') %>
```

Comment:
For accessibility, I think we'll want to be careful with what's the computed name of the button. Currently, I think it'd be computed as "Sign in with Login.gov logo". We had a similar issue in the brochure site implementation of a button like this, addressed in [LG-5296](https://cm-jira.usa.gov/browse/LG-5296).

I think there's a few different ways you could address this:

- Changing the image `alt` to just "Login.gov"
- Setting an `aria-label` on the button
- Using screen-reader-only text with a decorative image

e.g.

```html
<button>
  Sign in with <span class="usa-sr-only">Login.gov</span>
  <img src="..." alt="">
</button>
```